apps/schools/models/assessments.py

```diff
@@ -162,8 +163,8 @@ class AnswerStudent(models.Model):
             blank=True, null=True)
     last_modified_by = models.ForeignKey(User, blank=True, null=True,
             related_name='last_modified_answer_student')
-    flexi_data = models.CharField(max_length=30, blank=True, null=True)
-    active = models.IntegerField(blank=True, null=True, default=2)
+    flexi_data = models.CharField(max_length=30, blank=True, null=True) #Shivangi, is this needed?
```

Comment:
It was to enter some different kind of assessment data. I cant remember the logic behind it though. Might be for the data that is to be entered everyday for someone like akshay patra etc.